lesson_14/interfaces.py

```diff
@@ -0,0 +1,38 @@
+from lesson_14.handlers import JsonHandler, TxtHandler
+
+
+class FileWorker:
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.file_type = self.__file_type_determining()
+
+    def __file_type_determining(self):
+        if self.file_path.endswith('.json'):
+            return 'json'
+        elif self.file_path.endswith('txt'):
+            return 'txt'
+        else:
+            raise ValueError('Unsupported filetype!')
+
+    def read(self):
+        if self.file_type == 'json':
```

Comment:
You can use class dict here instead of `if` operator..
Like:
```Python
TYPES = {'json': JsonHandler, ...}
```
and here just get handler from dict by type